sklearn/svm/src/liblinear/linear.cpp

```diff
@@ -1701,10 +1705,9 @@ static int solve_l1r_l2_svc(
 // solution will be put in w
 //
 // See Yuan et al. (2011) and appendix of LIBLINEAR paper, Fan et al. (2008)
-
-#undef GETI
-#define GETI(i) (y[i]+1)
-// To support weights for instances, use GETI(i) (i)
+#undef GETC
+#define GETC(i) C[y[i]+1]*sample_weight[i]
+// To support weights (sample_weight*class_weight) for instances, use GETC(i) (i)
```

Comment:
If you're renaming `GETI` anyway, why not rename it to something intelligible like `SAMPLE_WEIGHT`?